core/services/workflows/cmd/cre/utils.go

```diff
@@ -193,6 +193,15 @@ func NewFakeCapabilities(ctx context.Context, lggr logger.Logger, registry *capa
 	}
 	caps = append(caps, fakeConsensus)
 
+	fakeConsensusNoDAG, err := fakes.NewFakeConsensusNoDAG(lggr)
```

Comment:
I think this needs to be wrapped in a consensusserver similar to L180 of this for HTTP

Reply:
You're right, it would make the capability fake implement Simple() instead of Execute(). Let me change it.